frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.scss

```diff
@@ -15,3 +15,11 @@
 .co-restore-pvc-modal__input {
   margin: var(--pf-global--spacer--md) 0;
 }
+
+.co-restore-pvc-modal__helpblock {
+  color: var(--pf-global--danger-color--100);
+}
```

Comment:
You might want to consider `has-error` which is used at most places in console for errors